inc/customizer.php

```diff
@@ -363,6 +363,40 @@ function hale_customize_register( $wp_customize ) {
 		)
 	);
 
+	$wp_customize->add_setting(
+		'heading_colour',
+		array(
+			'default'           => '#0b0c0c',
+			'sanitize_callback' => 'sanitize_hex_color',
```

Comment:
This sanitize callback will bean only hex values are allowed - I think this is a safer approach

Reply:
I agree keeping it to hex is good approach